build.sbt

```diff
@@ -0,0 +1,28 @@
+val appName = "address-lookup-performance-tests"
+val appVersion = "0.1.0-SNAPSHOT"
+
+lazy val root = (project in file("."))
+  .enablePlugins(GatlingPlugin)
+  .enablePlugins(CorePlugin)
+  .enablePlugins(JvmPlugin)
+  .enablePlugins(IvyPlugin)
+  .enablePlugins(SbtAutoBuildPlugin)
+  .settings(
+    organization := "uk.gov.hmrc",
+    name := appName,
+    version := appVersion,
```

Comment:
is there a reason we don't collapse the name and version down into this section?  What do we get from having them defined at the top of the file?